internal/pkg/pkg.go

```diff
@@ -0,0 +1,56 @@
+// Package pkg defines methods and heuristics for sanitizing package output.
+package pkg
+
+import (
+	"github.com/replit/upm/internal/api"
+)
+
+func SortNoop(query string, ignoredPackages []string, packages []api.PkgInfo) []api.PkgInfo {
+	return packages
+}
+
+func makeLoweredHM(normalizePackageName func(api.PkgName) api.PkgName, names []string) map[api.PkgName]bool {
+	// Build a hashset. struct{}{} purportedly is of size 0, so this is as good as we get
+	set := make(map[api.PkgName]bool)
+	for _, pkg := range names {
+		normal := normalizePackageName(api.PkgName(pkg))
+		set[normal] = true
+	}
+	return set
+}
+
+func SortPrefixSuffix(normalizePackageName func(api.PkgName) api.PkgName, query string, ignoredPackages []string, packages []api.PkgInfo) []api.PkgInfo {
+	ignoredPackageSet := makeLoweredHM(normalizePackageName, ignoredPackages)
```

Comment:
I thought we didn't need this anymore.
```suggestion
	ignoredPackageSet := make(map[string]bool)
```

Reply:
@airportyh Still need to run `normalizePackageName` on the packages, I can in-line though if you'd prefer

Reply:
I see the approval -- Thank you!